### DISCUSSION TITLE
Mutant Crops

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1127] [Art]
Moving Github issue to Discussion: https://github.com/sunflower-land/sunflower-land/issues/555

This is a request for hundreds of variations of Mutant Crops. Each crop will be unique so we need community submissions to make sure we have enough art! ❤️ 

When players farm, they will have a very small chance of discovering a mutant crop. These will be NFTs and ideally we want each one to be a unique design - hence we are looking for the community to submit hundreds of different variants 👏 

**Requirements**

- Art must be original.
- Similar dimensions to current crops in the game
- Include your Farm ID and social media handle on your submission

-https://github.com/sunflower-land/sunflower-land/blob/main/docs/ART_CONTRIBUTING.md

**Examples**

So far we have these amazing examples:

https://github.com/sunflower-land/sunflower-land/issues/507
https://github.com/sunflower-land/sunflower-land/issues/514

<img width="770" alt="Screen Shot 2022-04-15 at 5 21 15 pm" src="https://user-images.githubusercontent.com/11745561/163538649-40c4dda0-4d06-4b21-9b8d-ae515c690d78.png">

 (@AKCH1N)

**Comment by FarmerKuenster** [comment]
goofy idea I came up for mutant crop idea, sneeky eyed crops! 
I will update this with the other crops as I get them finished. 
SUNFLOWER
![sunflower-mutant-big-eyesl](https://user-images.githubusercontent.com/104322749/178155380-50bc2d72-8eaf-42d2-bf31-c8e27524bb28.gif)
![sunflower-mutant-big-eyes-large](https://user-images.githubusercontent.com/104322749/178155382-da4a5531-db99-48d5-af84-9bc605495796.gif)

CAULIFLOWER
![cauliflower-mutant-big-eye](https://user-images.githubusercontent.com/104322749/178126137-3b888206-3cfb-42be-bdab-32075ab97fa3.gif)
![cauliflower-mutant-big-eye-LARGE](https://user-images.githubusercontent.com/104322749/178126140-d472a08a-2b82-4591-a082-8b0c18ad26cb.gif)

PUMPKIN
![pumpkin-mutant-big-eyes](https://user-images.githubusercontent.com/104322749/178126329-b0eb6108-76d1-40c6-bead-cfb689d3c977.gif)
![pumpkin-mutant-big-eyes-LARGE](https://user-images.githubusercontent.com/104322749/178126334-3214ef4c-ee70-4ad6-8eac-0d9290557d8b.gif)

CABBAGE
![CABBAGE-MUTANT-BIGEYES-SMALL](https://user-images.githubusercontent.com/104322749/178155670-cb57f583-b425-4684-bb48-20c9ec8bf1e1.gif)
![cabbage-mutant-big-eyes-LARGE](https://user-images.githubusercontent.com/104322749/178155675-e00480f2-c18f-45d7-ae7d-56d4f3b7d6b9.gif)

WHEAT
![wheat-mutant-bigeyes-SMALL](https://user-images.githubusercontent.com/104322749/183722954-a2a4df6f-6318-48f2-9319-7a8f7a893ba2.gif)
![wheat-mutant-bigeyes-LARGE500](https://user-images.githubusercontent.com/104322749/183723416-f6ada160-7527-4ee4-9ff8-dd8862b11eed.gif)



discord: farmerkunester #1626
farm id: 32391

**Comment by JcEii** [comment]
Here are my own cute mutant crops. I finished this in a day because I was busy doing some interesting stuff...

**(Updated as of July 12, 2022 - V3)**

### Sample Image:
![non-animated](https://user-images.githubusercontent.com/105348036/178434776-42b88537-3628-458e-aa1d-229435e9917d.png)

### Images(Minified and Sample Image):
**Sunflower:**
![animatedsunflower](https://user-images.githubusercontent.com/105348036/178187196-f1057813-2461-482b-9209-718f2d5944c2.gif)
![Sunflower](https://user-images.githubusercontent.com/105348036/178187838-8f87e522-3492-4ba6-91b2-155c62a59804.gif)

**Potato:**
![animatedpotato](https://user-images.githubusercontent.com/105348036/178187208-9b240185-0e71-42e9-a226-4d615e328c71.gif)
![potato](https://user-images.githubusercontent.com/105348036/178187844-4a6e68d6-e50d-4907-b047-b7c4466286c4.gif)

**Pumpkin:**
![animatedpumpkin](https://user-images.githubusercontent.com/105348036/178187514-6a6acdd0-f03c-48bd-afd2-bd62fe8561f8.gif)
![Pumpkin](https://user-images.githubusercontent.com/105348036/178187855-07eb5246-4c39-4a0c-b0cf-088ea1e558d7.gif)
![Pumpkin1](https://user-images.githubusercontent.com/105348036/178187857-ab83b967-924a-478e-9ebd-b9979df83fe5.gif)

**Carrot:**
![animatedcarrot](https://user-images.githubusercontent.com/105348036/178434812-50b61c62-bc50-4568-adb2-15823937bbb2.gif)
![Carrot2](https://user-images.githubusercontent.com/105348036/178434825-08e0cb4c-b60d-4a4a-b58f-cae1691f54bd.gif)
![Carrot](https://user-images.githubusercontent.com/105348036/178434827-ea3c9f34-2463-4321-9b58-f7884f9135ab.gif)

**Cabbage:**
![animatedcabbage](https://user-images.githubusercontent.com/105348036/178187245-454135df-536c-4ff7-bf79-477055f61ce4.gif)
![Cabbage](https://user-images.githubusercontent.com/105348036/178187882-dadd29c5-b5f5-4761-bce1-1abe73d7b786.gif)
![Cabbage2](https://user-images.githubusercontent.com/105348036/178187883-0dd0e621-5d1a-4017-a01b-6a2326b13a9c.gif)

**Beetroot:**
![animatedbeetroot](https://user-images.githubusercontent.com/105348036/178187254-6c54af0a-86db-4723-877a-b260c9af3b64.gif)
![Beetroot](https://user-images.githubusercontent.com/105348036/178187887-6a7da060-0e7a-4dbf-b3ca-03b442687253.gif)

**Cauliflower:**
![animatedcaulit](https://user-images.githubusercontent.com/105348036/178187268-d4330f4e-5379-4ca1-834a-e5b4ccd3dff9.gif)
![Cauiflower](https://user-images.githubusercontent.com/105348036/178187892-2b2f8d43-c85c-4b94-9497-d4d51bf46c8e.gif)
![Cauiflower2](https://user-images.githubusercontent.com/105348036/178187894-0b8bf2ee-f647-46c7-9e44-09d6b4161f86.gif)

**Parsnip:**
![animatedParsnip](https://user-images.githubusercontent.com/105348036/178434886-c81eb078-6098-49ee-b7e3-cca3b4ce08f1.gif)
![Parsnip](https://user-images.githubusercontent.com/105348036/178434905-c65f2e8b-22fb-42dd-881e-13ce35777e3c.gif)


**Radish:**
![animatedRadish](https://user-images.githubusercontent.com/105348036/178187291-782d4aad-a4e6-4183-9c9e-6e84700e3798.gif)
![Radish](https://user-images.githubusercontent.com/105348036/178187922-078c8c8b-b063-4e5b-b45b-f5dbdcc8c3d4.gif)
![Radish2](https://user-images.githubusercontent.com/105348036/178187923-849f4f79-d0b1-4651-9190-38233b96bcdf.gif)

**Wheat:**
![animatedWheat](https://user-images.githubusercontent.com/105348036/178434939-81e21d5f-3296-4422-a7e6-3fa447f6c31d.gif)
![Wheat](https://user-images.githubusercontent.com/105348036/178434977-e5101155-b499-47dc-9687-b8c62d2f27c6.gif)

### Aseprite Files:
[Mutant Crop by Jc Eii.zip](https://github.com/sunflower-land/sunflower-land/files/9079761/Mutant.Crop.by.Jc.Eii.zip)
[Mutant Crop V2.zip](https://github.com/sunflower-land/sunflower-land/files/9080871/Mutant.Crop.V2.zip)
[Mutant Crop v3 Reanimated crops.zip](https://github.com/sunflower-land/sunflower-land/files/9090641/Mutant.Crop.v3.Reanimated.crops.zip)

Discord: Jc Eii # 3727
Farm ID: 15747

**Reply by MrSnorch** [reply]
Parsnip is so cute (❤ω❤)

**Comment by Alexcnder** [comment]
### **The Spider Collection**

![Preview](https://user-images.githubusercontent.com/87881787/180605064-0bb4c471-13df-436c-b552-f17f26b5ac75.gif)


![Cauliflower](https://user-images.githubusercontent.com/87881787/180605076-7d6d3ea7-2e22-4d84-8d9f-90d72e3935bb.gif)![Parsnip](https://user-images.githubusercontent.com/87881787/180605077-912b199e-48dc-4dd8-b843-0f3e145a2028.gif)![Potato](https://user-images.githubusercontent.com/87881787/180605078-71b059a7-0531-4de1-9bae-145e28d92388.gif)![Pumpkin](https://user-images.githubusercontent.com/87881787/180605079-df660092-0297-4201-a901-b375cde00604.gif)![Radish](https://user-images.githubusercontent.com/87881787/180605081-2e40e13f-5d42-4a9d-ac9e-f2c29a965987.gif)![SUnflower](https://user-images.githubusercontent.com/87881787/180605084-5698c637-0544-4c68-a1f7-eaab4d6c1fd5.gif)![Wheat](https://user-images.githubusercontent.com/87881787/180605085-a2999b27-2b9c-41dc-b66f-3f6e66bbe3bf.gif)![Beetroot](https://user-images.githubusercontent.com/87881787/180605086-37329855-60c0-4dfc-a01d-2176c14ff5c5.gif)![Cabbage](https://user-images.githubusercontent.com/87881787/180605087-84eb2262-e07e-40df-996c-d3728fbe6f71.gif)![CArrot](https://user-images.githubusercontent.com/87881787/180605088-9cb8ce6a-84dd-4337-9a88-11c340f3e77d.gif)

Twitter: Baaiex
Farm ID: 58

**Comment by Kiulg** [comment]
### Carrotdrile ![carrotdrile](https://user-images.githubusercontent.com/18270876/180628349-1c468eb9-9277-4774-bcc4-92a096bc93a1.png)
![carrotdrileb](https://user-images.githubusercontent.com/18270876/180628494-ea91b736-98d4-446d-969c-576340d82546.png)

### Pumpktopus ![plant (1)](https://user-images.githubusercontent.com/18270876/182761282-1a4b66fd-63ba-48eb-a197-1b298420b668.gif)
![pulp](https://user-images.githubusercontent.com/18270876/182761312-e61ba7e9-00e4-4cf8-8d00-6af76cded146.gif)


### Crabbage ![crabbage](https://user-images.githubusercontent.com/18270876/180628542-9c9c1d8f-9c27-473e-b009-82f2b58514f1.gif)
![crabbageb](https://user-images.githubusercontent.com/18270876/180628546-284078e9-1e21-4c7d-91a8-4c91156c3dab.gif)
 
### Beetreoot ![Bee](https://user-images.githubusercontent.com/18270876/184793928-e43af6d3-59bd-4064-9017-2454a9877c7f.gif)
![Bee2](https://user-images.githubusercontent.com/18270876/184793940-2f5ee4eb-2ee9-4c79-b9ad-a65cc8311242.gif)


### Hedgewheat ![hedgewheat](https://user-images.githubusercontent.com/18270876/180628553-9213cc1d-09e3-45b7-b95c-546350374853.gif)
![hedgewheatb](https://user-images.githubusercontent.com/18270876/180628554-b9c80e6f-22cb-441c-958b-28332a95976d.gif)

I'm going to do the rest if I get nice ideas xD

**Comment by On1el** [comment]
Let me know if you need any shadows
![АФИША](https://user-images.githubusercontent.com/104461938/180893364-73e49b16-184b-4ef6-9a59-392565a527f8.gif)
![цветная-капуста](https://user-images.githubusercontent.com/104461938/181030207-d2fd84f5-842c-4eee-87e6-30ca50443e82.gif)
![Sunflower](https://user-images.githubusercontent.com/104461938/181030240-81fcb7b1-8244-4be0-ac63-dd53cb954528.gif)
![капуста](https://user-images.githubusercontent.com/104461938/181030245-42a00f8f-5399-4249-97fb-6fdebe3b5d26.gif)
![морковь](https://user-images.githubusercontent.com/104461938/181030255-160f256f-f460-4c9d-ad4b-430837a09e02.gif)
![паршнып](https://user-images.githubusercontent.com/104461938/181030265-d44860d2-a520-4fc0-9bf0-701a6ee88855.gif)
![Пшено](https://user-images.githubusercontent.com/104461938/181030270-87baf524-128d-4c93-ade3-ddbb1f5be328.gif)
![редиска](https://user-images.githubusercontent.com/104461938/181030281-9734bbca-871f-405f-b3ef-58d0706c6a01.gif)
![свекла](https://user-images.githubusercontent.com/104461938/181030287-b4413dae-1aff-495d-a6f1-cf235ed0de3a.gif)
![тыква](https://user-images.githubusercontent.com/104461938/181030294-57c7d315-646b-4d25-bfd0-829e77faba39.gif)
![картоха](https://user-images.githubusercontent.com/104461938/181371821-e3991816-a717-4d5d-aa7f-1c08f240fdd5.gif)

**Reply by adamhannigan** [reply]
These are super cute! It makes me want to collect them all ❤️

**Reply by On1el** [reply]
![цветная-капуста](https://user-images.githubusercontent.com/104461938/181596458-19ae11b2-0f1b-4634-9f3e-53c48d41566d.gif)
![картошка](https://user-images.githubusercontent.com/104461938/181596462-c48afc78-d9b6-4105-9812-f7871169849d.gif)
![морковь](https://user-images.githubusercontent.com/104461938/181596464-0b539dbe-136b-4f36-b05b-a679478d108e.gif)
![паршнып](https://user-images.githubusercontent.com/104461938/181596466-a62c97d0-91b4-4a67-8497-5012d357991e.gif)
![подсолнух](https://user-images.githubusercontent.com/104461938/181596470-9ddfda1c-3087-4ec3-9d7d-191500e052f1.gif)
![пшено](https://user-images.githubusercontent.com/104461938/181596473-3099a417-91f6-4fe2-b316-dd6b774db4da.gif)
![редиска](https://user-images.githubusercontent.com/104461938/181596475-d1d1a3bd-6f91-44ca-8fb0-14c81aab0912.gif)
![свекла](https://user-images.githubusercontent.com/104461938/181596479-c5f461d5-4b92-43a7-9ae3-b58a459d9960.gif)
![тыква](https://user-images.githubusercontent.com/104461938/181596480-0f208cd8-c488-4c41-a73a-92d4c02cf152.gif)
![редиска](https://user-images.githubusercontent.com/104461938/181596551-328af2e9-ead0-4677-9709-ebd3b9cfabc0.gif)
![свекла](https://user-images.githubusercontent.com/104461938/181596554-6a114409-b310-4398-ae68-e8124b949668.gif)
![тыква](https://user-images.githubusercontent.com/104461938/181596555-aca7819a-a32f-4d47-8a90-2b54e8efc5b1.gif)
![цветная-капуста](https://user-images.githubusercontent.com/104461938/181596556-72c62b47-b375-4153-832e-fbb9db4e7f81.gif)
![капуста](https://user-images.githubusercontent.com/104461938/181596558-8384c1d1-4292-4b2c-a268-5bd59eda275f.gif)
![картоха](https://user-images.githubusercontent.com/104461938/181596559-beaed738-7b49-4406-9f3c-068c0755a467.gif)
![морковь](https://user-images.githubusercontent.com/104461938/181596561-6501c3bb-c5aa-470b-b414-d507d615fce3.gif)
![паршнып](https://user-images.githubusercontent.com/104461938/181596563-033ffefc-1b74-4cce-8098-7d631bbfaa41.gif)
![подсолнух](https://user-images.githubusercontent.com/104461938/181596564-26ae809a-3ae1-4fe2-9505-76bf9b137e64.gif)
![подсолнух2](https://user-images.githubusercontent.com/104461938/181596565-edebd01e-336d-4fe8-bd01-864788ed7279.gif)
![пшено](https://user-images.githubusercontent.com/104461938/181596568-e184b823-1520-417c-ad4d-2918c6ae7fb7.gif)

**Comment by Whaitte** [comment]
**Rich sunflower**
 ![sunflower nerd (3).gif](https://user-images.githubusercontent.com/104186329/188044278-9869e5bf-4660-43bf-9562-949fabd75d26.gif)![sunflower nerd (2).gif](https://user-images.githubusercontent.com/104186329/188044334-0a86dbb2-2883-4e39-a20f-8c12250a458c.gif)
**2nd variety**
![Untitled 08-27-2022 09-58-54 (2).gif](https://user-images.githubusercontent.com/104186329/188044477-9d6bbeda-57c7-4608-bd01-da91fe8050ed.gif)
![Untitled 08-27-2022 09-58-54.gif](https://user-images.githubusercontent.com/104186329/188044495-1f0af899-9b97-4e3e-9a69-0ff8dff14798.gif)
**3rd variety**
![Untitled 08-27-2022 04-04-17 (4).gif](https://user-images.githubusercontent.com/104186329/188045317-54900275-ed8b-435e-9807-d5eacb7f4854.gif)![Untitled 08-27-2022 04-04-17 (2).gif](https://user-images.githubusercontent.com/104186329/188045342-7c8c9967-9da2-43f1-8fc4-7ae4286664b4.gif)


Farm ID: #919
Farm address: 0xBBb40e3E4359c7af0254728ab52A4c39a14f5c08
Discord: whaitte#3465

**Comment by shinooon** [comment]
Mutant Radish Preview
![mutant radish prev](https://user-images.githubusercontent.com/109294090/198954968-8890fd79-6d71-48f2-ae08-967a84721eb5.gif)
![mutant radish](https://user-images.githubusercontent.com/109294090/198954996-604cc5fa-3e04-4834-8bba-8754f4642c8a.gif)
Mutant Twin Sunflower
![mutant sunflower prev](https://user-images.githubusercontent.com/109294090/198955061-26250ef7-9c12-40c6-84f6-86e76c84e38f.gif)
![mutant sunflower](https://user-images.githubusercontent.com/109294090/198955070-f81755ff-36a6-4d96-8027-3cc388fbf11c.gif)

farm id: 51738
discord: fam#4931 (shinon)
